packages/http-client-python/emitter/src/http.ts

```diff
@@ -1,5 +1,6 @@
 import {
   SdkBasicServiceMethod,
+  SdkBodyModelPropertyType,
```

Comment:
Will the PR cause SDK diff?

Reply:
Nope. I don't think we have test on this in cadl-ranch. I just tested it locally.